source/commands/create_command.py

```diff
@@ -0,0 +1,45 @@
+import os, sys,json
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from commands.abstract_command import AbstractCommand
+from commands.keys import *
+from output.exceptions import *
+
+
+class CreateCommand(AbstractCommand):
+    def __init__(self , schema_path):
+        self.schema_path = schema_path
+        self.validate(self)  
+    
+    @staticmethod    
+    def validate(self):
```

Comment:
How this method is static and take self ?

Reply:
I removed the ```@staticmethod``` and it takes self because I need it in the ```.schema_path```

Reply:
That is why I resolved it.
I don't clearly understand what you mean by static but when I searched about it I found that the static method is a method that is bound to the class and not the object of the class and that this method can’t access or modify the class state.

Reply:
No it shouldn’t be instance method. The question here how to decide to make the method static or instance method?
- static method doesn’t change in the instance variable
- Static method doesn’t access instance variable.
- Static method doesn’t change in object state ( this is same point #1)


- This method is validate which expects to validate things and doesn’t change in the object state, so it shouldn’t set data object.

that’s why I recommend to return the data object. in this case, it won’t change in any instance variable, so it should be static 
Self.schema = GreateCommand.validate